examples/ThreeD_Plots.jl

```diff
@@ -1,6 +1,6 @@
 using GLMakie
-using AbstractPlotting.Colors
-AbstractPlotting.inline!(true)
+using AbstractPlotting
+AbstractPlotting.inline!(true) 
```

Comment:
careful with changes that are not really changes.

Reply:
I changed it because with just the `.Colors` I had a `ERROR: LoadError: UndefVarError: AbstractPlotting not defined`

Reply:
I discussed this error in a Julia forum and was told to `use` the whole package.
Concerning the vscode file, that was supposed to be ignored with `gitignore`, sorry for the confusion.